.github/workflows/base-ci.yaml

```diff
@@ -73,5 +73,4 @@ jobs:
         assert str(RDKIT_AVAILABLE).lower() == 'true'
 
     - name: Run tests
-      run: |
-        python -m pytest -n 4 -v --cov=openff/nagl --cov-config=setup.cfg --cov-append --cov-report=xml --color=yes openff/nagl/
```

Comment:
Small quirk: since `setup.cfg` exists, this won't throw an error (until that changes, anyway) but it won't pick up any actual configs as they're in `pyproject.toml`. A leftover detail from #169